### ISSUE TITLE
Not Loading Custom QMK Keymap  JSON

### ISSUE THREAD

**Issue by otaviocc** [qmk/qmk_configurator#752]
When I try to load a valid custom JSON, it gives me the follow screen, no matter if loading from the disk, or from the cloud.

<img width="1380" alt="Screen Shot 2020-06-18 at 2 50 47 PM" src="https://user-images.githubusercontent.com/139272/85035837-e63fe280-b183-11ea-9e57-f93f3f81835d.png">

I've tried Firefox, Firefox Private Mode, Google Chrome and Chrome Private Mode. Same result in all of them.

**Comment by yanfali** [comment]
@noroadsleft my guess is the default still references a layout that no longer exists LAYOUT_preonic_grid. That's why it fails after loading the default.

**Comment by noroadsleft** [comment]
Yeah, apparently something in the API has changed and it no longer picks up the aliases for the Preonic.

Gimme a minute...

**Comment by noroadsleft** [comment]
Fixed by 0b035229185fca4d04ad1403c713348d5ce4d427.
